kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -3067,7 +3067,8 @@ object KyuubiConf {
         "<li>mysql: For establishing MySQL connections.</li> " +
         "<li>phoenix: For establishing Phoenix connections.</li> " +
         "<li>postgresql: For establishing PostgreSQL connections.</li>" +
-        "<li>starrocks: For establishing StarRocks connections.</li>")
+        "<li>starrocks: For establishing StarRocks connections.</li>" +
+        "<li>clickhouse: For establishing clickhouse connections.</li>")
```

Comment:
seems we forgot to add Impala, could u please open a new PR to add Impala case?

Reply:
ok